### ISSUE TITLE
Add cloud object store support (S3 etc)

### ISSUE THREAD

**Issue by andygrove** [andygrove/bdt#10]

**Comment by YuriyGavrilov** [comment]
Maybe add support of storj network through uplink https://github.com/storj/uplink

**Comment by trungda** [comment]
Seems this is still a nice-to-have feature? I'd like to take a stab at this to understand more the datafusion ecosystem.

**Comment by andygrove** [comment]
> Seems this is still a nice-to-have feature? I'd like to take a stab at this to understand more the datafusion ecosystem.

Yes, it is. DataFusion supports reading from S3 and other object stores already, so hopefully this is not too hard to add.

**Comment by ForeverAngry** [comment]
@trungda I'm interested in this feature. Is this something you have solved?

**Comment by trungda** [comment]
> @trungda I'm interested in this feature. Is this something you have solved?

No I haven't yet. It's on my list but I don't have time to get to it in the next month. Feel free to jump into it?
